legend-engine-ide-lsp-server/src/main/java/org/finos/legend/engine/ide/lsp/server/LegendTextDocumentService.java

```diff
@@ -95,56 +94,67 @@ class LegendTextDocumentService implements TextDocumentService
     @Override
     public void didOpen(DidOpenTextDocumentParams params)
     {
+        LegendServerGlobalState globalState = this.server.getGlobalState();
+        this.server.checkReady();
```

Comment:
I don't think it's necessary to call checkReady here, as that's done in getGlobalState.